File: AndroidApp&Library/Android/sampleandroidlib/src/main/java/com/ev/sampleandroidlib/LoggerStaticLibrary.java
package com.ev.sampleandroidlib;

public class LoggerStaticLibrary {

    public static String greetUser(String name) {
        return "Hi " + name + " Welcome to EdgeVerve !";
    }

    public  static  void logInLibrary(){
        System.out.println("You are inside Library!!! Method");
    }

    public  static  int addTwoNumbers(int num1, int num2) {
        return  num1 + num2;
    }
}

File: LoggerStaticLibrary/LoggerStaticLibrary/LoggerStaticLibrary.m
//
//  LoggerStaticLibrary.m
//  LoggerStaticLibrary
//
//  Created by Naresh on 08/05/20.
//  Copyright © 2020 Naresh. All rights reserved.
//

#import "LoggerStaticLibrary.h"

@implementation LoggerStaticLibrary

+(NSString *)greetUser:(NSString *)userName{
    return [NSString stringWithFormat:@"Hi %@ welcome to Edgeverve",userName];
}

+(void)logInLibrary {
    NSLog(@"You are inside Library!!! Method");
}

+(int)addTwoNumbers:(int)num1 :(int)num2 {
    return num1+num2;
}

@end
